apps/studio/src/lib/db/authentication/azure.ts

```diff
@@ -153,17 +172,43 @@ export class AzureAuthService {
     }
   }
 
+  private async getAzureCLIToken(options: AzureAuthOptions): Promise<AuthConfig> {
+    if(!options?.cliPath){
+      throw new Error('AZ command not specified')
+    }
+
+    const command = `${options?.cliPath} account get-access-token --resource ${globals.azSQLLoginScope} --output json`;
```

Comment:
`options.cliPath` needs to be properly escaped. Also I don't think you imported `execAsync`?

Reply:
To do proper escaping you should use `spawn`, that takes a command, and a list or arguments, that way the `cliPath` will be appropriately escaped.